service/storage_service.cc

```diff
@@ -2127,20 +2141,12 @@ class topology_coordinator {
                 switch (node.request.value()) {
                     case topology_request::join: {
                         assert(!node.rs->ring);
-                        auto num_tokens = std::get<join_param>(node.req_param.value()).num_tokens;
-                        // A node just joined and does not have tokens assigned yet
-                        // Need to assign random tokens to the node
-                        auto tmptr = get_token_metadata_ptr();
-                        auto bootstrap_tokens = dht::boot_strapper::get_random_bootstrap_tokens(
-                                tmptr, num_tokens, dht::check_token_endpoint::yes);
-
                         // Write chosen tokens through raft.
                         builder.set_transition_state(topology::transition_state::join_group0)
                                .with_node(node.id)
                                .set("node_state", node_state::bootstrapping)
-                               .del("topology_request")
-                               .set("tokens", bootstrap_tokens);
```

Comment:
So basically at some point of bootstrap process we will have a joining node without any tokens. Right?